File: task-manager-api/src/main/java/com/tm/dto/bean/task/TaskCompleteRequestDto.java
package com.tm.dto.bean.task;

/**
 * タスク完了リクエストDTO。
 */
public class TaskCompleteRequestDto {

    /**
     * タスクID
     */
    private String taskId;

    public String getTaskId() {
        return taskId;
    }
    public void setTaskId(String taskId) {
        this.taskId = taskId;
    }
}

File: task-manager-app/src/app/dto/interface/task-complete-request.dto.ts
/**
 * タスク完了リクエストDTOクラス。
 */
export class TaskCompleteRequestDto {

    /**
     * タスクID
     */
    private taskId: string;

    public getTaskId(): string {
        return this.taskId;
    }
    public setTaskId(taskId: string): void {
        this.taskId = taskId;
    }
}